lib/tapioca/compilers/symbol_table/symbol_generator.rb

```diff
@@ -655,6 +659,13 @@ def compile_method(tree, symbol_name, constant, method, visibility = RBI::Public
           tree << rbi_method
         end
 
+        sig { params(method: T.any(UnboundMethod, Method)).returns(String) }
+        def yard_docs(method)
+          Pry::Method.new(method).doc
+        rescue StandardError
```

Comment:
Some weird methods inside `pry` cause errors. Like, for example, they have a `SourceLocation#[](*)` method that has no receiver (I don't know how), and that breaks its own code. So, I think it's fine to leave this as a best effort and if we can't get the docs, just compile the method without it.

Reply:
Shouldn't we return `nil` instead to avoid creating comments with empty strings?

Reply:
We invoke `lines` on the empty string, which returns an empty array and then when we invoke `each` it won't perform any loops. This is basically what happens

```ruby
"".lines -> []
[].each {...} -> nothing happens
```